### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by iLynette**
In this project, I implemented the following;

- Used `formspree` on the contact form
- Used `HTML` and `CSS` best practices
- Used `figma`

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -282,3 +281,55 @@ h1 span {
   border-bottom: 8px;
   background-color: #ebf0ee;
 }
+
+.contact {
+  background: no-repeat 100% 100% url(./images/illustration-contact-form.png);
+  padding: 64px 24px 64px 24px;
+}
```
Comment (elmar8287) [inline style.css:293]: - [ ] Please, make sure that you use Flexbox to layout the elements.

File: style.css [hunk 2]
```diff
@@ -282,3 +281,55 @@ h1 span {
...
+  border-radius: 4px;
+  padding: 12px;
+  display: flex;
+  margin: auto;
+  color: white;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  text-align: center;
+}
```
Comment (elmar8287) [inline style.css:340]: - [ ] Please, consider the button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are styled as shown in the Figma template.

File: style.css (hunk 1 above)
Comment (elmar8287) [inline style.css:293]: - [ ] I also suggest to you to use 100% width for big screens

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi @iLynette ,

Great work on making the project requirements 👏 

There are still some requirements that aren't done yet, please work on it and ask for another review.
      
 ## Required Changes ♻️

- [ ] Please, consider comments under review

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: style.css (hunk 1 above)
Comment (iLynette) [reply style.css:293]: Hi @elmar8287 when you say big screen do you mean desktop version?

Comment (elmar8287) [reply style.css:293]: Dear @iLynette like another device with a bigger display, as a tablet, notebook...

Comment (iLynette) [reply style.css:293]: Okay thank you @elmar8287 I'm working on it

**Review by Meltrust — APPROVED** [review]
Hi @iLynette, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**